### PR TITLE
Update [FR] BanchoBot

### PR DESCRIPTION
**Pull request by DoryanR**

### CODE REVIEW

File: wiki/BanchoBot/fr.md [hunk 1]
```diff
@@ -1,40 +1,191 @@
...
+
+*Remarque: vous pouvez également utiliser la carte d’utilisateur pour signaler un utilisateur.*
+
+```
+!REPORT <utilisateur> <raison>
+```
+
+*Remarque: lorsque vous entrez le nom d'utilisateur, remplacez les espaces par des underscores (`_`).*
+
+Envoie un rapport à [L'Equipe Mondiale de Modération](/wiki/Global_Moderation_Team) ou les [Modérateurs de Langues](/wiki/Language_Moderators).
```
Comment (Pachiru) [inline wiki/BanchoBot/fr.md]:
```suggestion
Envoie un rapport à [l'équipe de modération](/wiki/Global_Moderation_Team) ou les [modérateurs par langues](/wiki/Language_Moderators).
```

File: wiki/BanchoBot/fr.md [hunk 2]
```diff
@@ -1,40 +1,191 @@
...
+BanchoBot: Les utilisateurs chinois s'il vous plaît cliquez sur #chinese Pour entrer dans le canal chinois pour la communication.
+
+BanchoBot: #osu s’agit d’un canal en anglais seulement. Si vous continuez de parler chinois à #osu après avoir reçu ce message, l’administrateur a le droit d'appliquer des sanctions.
+
+Tama: ok
+```
+
+### REPORT
+
+*Notice: Si vous souhaitez signaler un modérateur, envoyez un courrier électronique à [support@ppy.sh](mailto:support@ppy.sh) pour les plaintes du modérateur.*
```
Comment (Pachiru) [inline wiki/BanchoBot/fr.md]:
```suggestion
*Notice: Si vous souhaitez signaler un modérateur, envoyez un e-mail à [support@ppy.sh](mailto:support@ppy.sh).*
```

**Comment by DoryanR** [comment]
> check how EN groups the sentences into paragraphs, it looks like you split some paragraphs into many small ones

I do this because I think it's easier to read, otherwise it's a big block of text that's hard to read.

If this is really a problem I can redo big blocks of text, but if it is possible I would like to leave like that, for the comprehension of the article.

**Comment by cl8n** [comment]
related ideas should be in the same paragraphs, that's what paragraphs are for after all. having a lot of space between sentences implies that they aren't closely related, which isn't true in this case

also just for future reference, if you change any formatting in translations, make sure to update EN as well (and change/outdate other translations if necessary). for this one I think EN is good though and u should change FR to match EN

**Comment by cl8n** [comment]
can merge after @Pachiru rechecks

**Pull request merged**